lib/commands/sign-jwt.js

```diff
@@ -1,35 +1,34 @@
-import jose from 'node-jose';
-import {v4 as uuid} from 'uuid';
+import crypto from 'crypto';
+import fs from 'fs/promises';
 import chalk from 'chalk';
-import fs from 'fs';
-import querystring from 'querystring';
-import fetch from 'node-fetch';
+import {SignJWT} from 'jose';
+import {v4 as uuid} from 'uuid';
 
 export default async function signJWT(clientId, privateKeyPath) {
     // Obtain an access token to call banno apis
     const enterpriseOidcTokenUri = 'https://banno.com/a/oidc-provider/api/v0/token';
 
     const jwtPayload = {
-        jti: uuid(),
-        aud: enterpriseOidcTokenUri,
-        sub: clientId,
-        iss: clientId,
-        exp: Date.now() + 60 * 1000
+      jti: uuid(),
+      aud: enterpriseOidcTokenUri,
+      sub: clientId,
+      iss: clientId,
+      exp: Date.now() + 60 * 1000
     };
 
     process.stdout.write(`${chalk.green('Client JWT payload:')} ${JSON.stringify(jwtPayload, null, 2)}\n`);
 
-    const privateKey = await fs.promises.readFile(privateKeyPath, {encoding: 'utf8'});
-
-    const jwkKeystore = jose.JWK.createKeyStore();
-    await jwkKeystore.add(privateKey, 'pem');
-    const jwkKey = jwkKeystore.all({use: 'sig', alg: 'PS256'})[0];
+    const privateKeyContents = await fs.readFile(privateKeyPath, {encoding: 'utf8'});
+    const privateKey = crypto.createPrivateKey({
+      key: privateKeyContents,
+      format: 'pem',
+      encoding: 'utf8'
+    });
```

Comment:
**praise:** One nice thing about this new jose library is that this signing code is a _lot_ easier to read and understand. Feels like the API choices in this new library emphasize readability in a way that node-jose doesn't. 👍

Reply:
We've moved most of our internal projects to this library. Both for readability reasons and for dependencies.